sgt-back-end/index.js

```diff
@@ -0,0 +1,212 @@
+const pg = require('pg');
+var express = require('express');
+var app = express();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+var middle = express.json();
+
+app.use(middle);
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+    from "grades"
+
+  `;
+
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200);
+      res.json(grades);
+
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred. '
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  if (!Number.isInteger(Number(req.body.score)) || req.body.score < 0 || req.body.score > 100 || req.body.score === '' || req.body.score === undefined || req.body.name === '' || req.body.name === undefined || req.body.course === '' || req.body.course === undefined) {
```

Comment:
If a line of code ends up being more than a screen or two wide, it's probably a good idea to break it up into multiple lines.